lib/router.js

```diff
@@ -45,18 +46,26 @@ router.get('/slashCommands/ci', (req, res) => {
 router.get('/slashCommands/ext', (req, res) => {
     const command = req.query && req.query.command;
     const text = req.query.text;
+    const update = req.query.update;
 
     if (command && command === '/ramal') {
-        const findExtension = text ?
-            extCommand.find.bind(extCommand, text) :
-            extCommand.find.bind(extCommand);
-
-        findExtension(out => {
-            res.send(out);
-        });
+        if (update && update === 'true' && config.extMode === 'SHEET') {
+            extCommand.update(text, out => {
+                res.send(out);
+            });
+        }else{
```

Comment:
It appears you have mixed code style throughout your code (sometimes you put spaces before blocks, sometimes you don't), differing from the existing code... I've added some more ESLint rules to help with that. Rebase your branch, `npm install`, then run `eslint . --fix` and it will fix it for you automatically ;)